packages/php-wasm/web/src/lib/web-php-endpoint.ts

```diff
@@ -89,6 +89,11 @@ export class WebPHPEndpoint implements IsomorphicLocalPHP {
 		return _private.get(this)!.php.mv(fromPath, toPath);
 	}
 
+	/** @inheritDoc @php-wasm/universal!IsomorphicLocalPHP.mv  */
```

Comment:
Should this be updated?

Reply:
https://github.com/WordPress/wordpress-playground/pull/846/files#diff-eeec0049f1f178834acbd06e83e370eb8044a650901eb4dd3f9189126a07460eR266